diff --git a/tools/n64splat/rareunzip.py b/tools/n64splat/rareunzip.py
new file mode 120000
index 0000000..0dfc8ea
--- /dev/null
+++ b/tools/n64splat/rareunzip.py
@@ -0,0 +1 @@
+../rareunzip.py
\ No newline at end of file
diff --git a/tools/n64splat/segtypes/mp3.py b/tools/n64splat/segtypes/mp3.py
new file mode 120000
index 0000000..a6fee62
--- /dev/null
+++ b/tools/n64splat/segtypes/mp3.py
@@ -0,0 +1 @@
+../../../patches/n64splat/mp3.py
\ No newline at end of file
diff --git a/tools/n64splat/segtypes/rzip.py b/tools/n64splat/segtypes/rzip.py
new file mode 120000
index 0000000..6e4aa49
--- /dev/null
+++ b/tools/n64splat/segtypes/rzip.py
@@ -0,0 +1 @@
+../../../patches/n64splat/rzip.py
\ No newline at end of file
